sitemap.php

```diff
@@ -310,6 +310,7 @@ protected function addRouteData($pages, $lang)
                 $include_lang = $this->multilang_skiplang_prefix !== $lang;
                 $location = $page->canonical($include_lang);
                 $url = $page->url(false, $include_lang);
+				$lastmod = isset($header->sitemap['lastmod']) ? strtotime($header->sitemap['lastmod']) : $page->modified();
```

Comment:
Would `!empty()` instead of `isset()` make more sense here? Empty string is still set 🤔

Reply:
I'm quite new to PHP, I don't know all the best practices and functions yet. I'll update it soon, thanks.